src/mono/wasm/runtime/memory.ts

```diff
@@ -114,3 +136,7 @@ export function getF32(offset: _MemOffset): number {
 export function getF64(offset: _MemOffset): number {
     return Module.HEAPF64[<any>offset >>> 3];
 }
+
+export function updateGlobalBufferAndViews(buffer: Buffer): void {
```

Comment:
we should give this a different name from the function it's replacing, since it doesn't call the original or do what the original did

Reply:
renamed to `afterUpdateGlobalBufferAndViews`